### PR TITLE
Docs/psql9.0 docs merge a

### PR DESCRIPTION
**Pull request by litzell**
Edits GPDB docs from sgml files updated by the first postgresql 9.0 merge 
https://github.com/greenplum-db/gpdb/pull/4528

Staged docs are here:

Commands: https://docs-litzell-psql90-merge-a.cfapps.io/600/ref_guide/sql_commands/sql_ref.html

psql: https://docs-litzell-psql90-merge-a.cfapps.io/600/utility_guide/client_utilities/psql.html

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/sql_commands/ALTER_TABLE.xml [hunk 1]
```diff
@@ -306,7 +327,11 @@ ALTER TABLE <varname>name</varname>
                                                 If the <codeph>ONLY</codeph> keyword is not used,
                                                 the operation will be performed on the named table
                                                 and any child table partitions associated with that
-                                                table.</pd>
+                                                table. Adding or dropping a column, or changing a
+                                                column's type in a parent or descendant table only
+                                                is not permitted. The parent table and its
+                                                descendents must always have the same columns and
+                                                types.</pd>
```
Comment (mkiyama) [inline gpdb-doc/dita/ref_guide/sql_commands/ALTER_TABLE.xml]: Should this be a note or a warning?

File: gpdb-doc/dita/ref_guide/sql_commands/SELECT.xml [hunk 2]
```diff
@@ -770,42 +780,79 @@ FETCH { FIRST | NEXT } [ <varname>count</varname> ] { ROW | ROWS } ONLY</codeblo
...
-          row-level lock(s) — the required <codeph>ROW SHARE</codeph> table-level lock is still
-          taken in the ordinary way. You can use the <codeph>NOWAIT</codeph> option of
-            <codeph>LOCK</codeph> if you need to acquire the table-level lock without waiting (see
-              <codeph><xref href="LOCK.xml#topic1" type="topic" format="dita"/></codeph>).
-            </p><p><codeph>FOR SHARE</codeph> behaves similarly, except that it acquires a shared
-          rather than exclusive lock on the table. A shared lock blocks other transactions from
-          performing <codeph>UPDATE</codeph>, <codeph>DELETE</codeph>, or <codeph>SELECT FOR
-            UPDATE</codeph> on the table, but it does not prevent them from performing
-            <codeph>SELECT FOR SHARE</codeph>.</p><p>If specific tables are named in <codeph>FOR
+            UPDATE</codeph> causes the rows accessed by the <codeph>SELECT</codeph> statement to be
```
Comment (mkiyama) [inline gpdb-doc/dita/ref_guide/sql_commands/SELECT.xml]: SELECT FOR UPDATE
NEW GUC for global deadlock detector. gp_enable_global_deadlock_detector
default is disabled. 
PR 
https://github.com/greenplum-db/gpdb/pull/6828

File: gpdb-doc/dita/ref_guide/sql_commands/ALTER_TABLE.xml (hunk 1 above)
Comment (litzell) [reply gpdb-doc/dita/ref_guide/sql_commands/ALTER_TABLE.xml]: I promoted the restriction to a note. Consolidation is good.

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_TABLE.xml [hunk 3]
```diff
@@ -328,6 +372,45 @@
           <pd>For information about unique constraint management and limitations, see <xref
               href="#topic1/section5" format="dita"/>.</pd>
         </plentry>
+        <plentry>
+          <pt>EXCLUDE [ USING <varname>index_method</varname> ] ( <varname>exclude_element</varname> WITH <varname>operator</varname> [, ... ] ) <varname>index_parameters</varname> [ WHERE ( <varname>predicate</varname> ) ]</pt>
+          <pd>The <codeph>EXCLUDE</codeph> clause defines an exclusion
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/sql_commands/CREATE_TABLE.xml]: Not necessary to address in this PR, but Greenplum does have additional limitations with regard to exclusion constraints.  You might want to see if any of this text can be easily modified to match our requirements...

File: gpdb-doc/dita/ref_guide/sql_commands/SELECT.xml (hunk 2 above)
Comment (litzell) [reply gpdb-doc/dita/ref_guide/sql_commands/SELECT.xml]: I linked to the new guc. Hopefully that PR will be merged first.

**Pull request merged**